File: src/main/java/com/netsensia/rivalchess/model/util/CommonUtils.java
package com.netsensia.rivalchess.model.util;

public class CommonUtils {

    private CommonUtils() {}

    public static boolean isValidRankFileBoardReference(final int n) {
        return (n >= 0 && n <= 7);
    }

    public static boolean isValidSquareReference(final int x, final int y) {
        return isValidRankFileBoardReference(x) && isValidRankFileBoardReference(y);
    }
}

File: src/main/java/com/netsensia/rivalchess/model/util/CommonUtils.kt
package com.netsensia.rivalchess.model.util

object CommonUtils {
    @kotlin.jvm.JvmStatic
    fun isValidRankFileBoardReference(n: Int): Boolean {
        return n in 0..7
    }

    fun isValidSquareReference(x: Int, y: Int): Boolean {
        return isValidRankFileBoardReference(x) && isValidRankFileBoardReference(y)
    }
}